metadata-ingestion/tests/integration/looker/expected_output.json

```diff
@@ -1,98 +1,98 @@
 [
-    {
-        "auditHeader": null,
-        "proposedSnapshot": {
-            "com.linkedin.pegasus2avro.metadata.snapshot.ChartSnapshot": {
-                "urn": "urn:li:chart:(looker,dashboard_elements.2)",
-                "aspects": [
-                    {
-                        "com.linkedin.pegasus2avro.chart.ChartInfo": {
-                            "customProperties": {},
-                            "externalUrl": null,
-                            "title": "",
-                            "description": "Some text",
+{
```

Comment:
What's the diff here?

Reply:
Mainly replacing `looker_views` with `looker`. The whitespace changes do obscure the diff, but it looked good when using `diff -w` to ignore those changes